doc/howto_release.md

```diff
@@ -271,15 +272,15 @@ To easily generate the entries for the trac Wiki release page, use the `git log`
 ```
 # get date of previous release from https://github.com/OSGeo/grass/releases
 # verify
-git log --oneline --after="2020-10-05" | tac
+git log --oneline --after="2021-10-10" | tac
 
 # prepare for trac Wiki release page (incl. PR trac macro)
-git log --oneline --after="2020-10-05" | cut -d' ' -f2- | sed 's+^+ * G78:+g' | sed 's+(#+(PR:+g' | sort -u
+git log --oneline --after="2021-10-10" | cut -d' ' -f2- | sed 's+^+ * G80:+g' | sed 's+(#+(PR:+g' | sort -u
 ```
 
 - store changelog entries in trac, by section:
-    - <https://trac.osgeo.org/grass/wiki/Release/7.8.x-News>
-    - <https://trac.osgeo.org/grass/wiki/Grass7/NewFeatures78>  <- add content of major changes only
+    - <https://trac.osgeo.org/grass/wiki/Release/8.0.x-News>
+    - <https://trac.osgeo.org/grass/wiki/Grass8/NewFeatures80>  <- add content of major changes only
 
```

Comment:
Shall we keep these two if the approach generating the "nice" changelog in github works out?

Reply:
Once it really works, we may change it. This is a "living" document...